tools/packaging/packaging.psm1

```diff
@@ -3092,105 +3092,152 @@ function Get-PackageVersionAsMajorMinorBuildRevision
 .SYNOPSIS
 Create a smaller framework dependent package based off fxdependent package for dotnet-sdk container images.
 
-.PARAMETER FxdPackagePath
+.PARAMETER Path
 Path to the folder containing the fxdependent package.
-
-.PARAMETER ReleaseTag
-Release tag to construct the package name.
 #>
-function New-DotnetSdkContainerFxdPackage {
-    [CmdletBinding(SupportsShouldProcess = $true)]
+function ReduceFxDependentPackage
+{
+    [CmdletBinding()]
     param(
-        [Parameter(Mandatory)] $FxdPackagePath,
+        [Parameter(Mandatory)] [string] $Path
+    )
 
-        [ValidatePattern("^v\d+\.\d+\.\d+(-\w+(\.\d+)?)?$")]
-        [ValidateNotNullOrEmpty()]
-        [Parameter(Mandatory)] $ReleaseTag,
+    if (-not (Test-Path $path))
+    {
+        throw "Path not found: $Path"
+    }
 
-        [Parameter(Mandatory)] $DestinationPath
-    )
+    ## Remove unnecessary files
+    $localeFolderToRemove = 'cs', 'de', 'es', 'fr', 'it', 'ja', 'ko', 'pl', 'pt-BR', 'ru', 'tr', 'zh-Hans', 'zh-Hant'
```

Comment:
What if there will new folder? Maybe not "remove" but "use/filter only necessary folder"?

Reply:
Can I take this as a bug fix after this PR?

Reply:
Yes.